src/main/java/gregtech/integration/forestry/mutation/MaterialMutationCondition.java

```diff
@@ -0,0 +1,64 @@
+package gregtech.integration.forestry.mutation;
+
+import gregtech.api.unification.material.Material;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.init.Blocks;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.text.translation.I18n;
+import net.minecraft.world.World;
+import net.minecraftforge.oredict.OreDictionary;
+
+import forestry.api.apiculture.IBeeHousing;
+import forestry.api.climate.IClimateProvider;
+import forestry.api.genetics.IAllele;
+import forestry.api.genetics.IGenome;
+import forestry.api.genetics.IMutationCondition;
+import forestry.core.tiles.TileUtil;
+import forestry.core.utils.Translator;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.apache.commons.lang3.StringUtils.capitalize;
+
+public class MaterialMutationCondition implements IMutationCondition {
+
+    private final Set<IBlockState> acceptedBlocks = new HashSet();
+    private final String displayName;
+
+    public MaterialMutationCondition(Material material) {
+        this.displayName = I18n.translateToLocal("gregtech.mutation.block_of") + " " + material.getLocalizedName();
```

Comment:
Instead of this, with the material name separate, you could have the lang entry as "Requires block of %s", and then just pass the material name into the lang call